frontend/src/components/SupportedAppTitle.tsx

```diff
@@ -11,33 +10,25 @@ type SupportedAppTitleProps = {
 };
 
 const SupportedAppTitle: React.FC<SupportedAppTitleProps> = ({ odhApp, showProvider = false }) => {
-  const supportedImageClasses = classNames('odh-card__supported-image', {
-    'm-hidden': !isRedHatSupported(odhApp),
-  });
   return (
     <CardTitle>
-      <Flex alignItems={{ default: 'alignItemsCenter' }} spaceItems={{ default: 'spaceItemsSm' }}>
-        <FlexItem>{odhApp.spec.displayName}</FlexItem>
-        <FlexItem>
-          {/* We still have some layout issue after applying removeFindDomNode
-          Use Bullseye layout to temp fix it, here is the PF issue to track 
-          https://github.com/patternfly/patternfly-react/issues/8277 */}
+      <span style={{ display: 'flex' }}>
```

Comment:
Hah -- you're going to need to explain this. Why are you using a `span` tag for `flex` when PF provides us a Flex component?